smarts/core/agent_interface.py

```diff
@@ -207,6 +207,11 @@ class AgentInterface:
     The choice of action space, this action space also decides the controller that will be enabled.
     """
 
+    vehicle_type: str = "car"
```

Comment:
How did you choose the word `"car"`, is this the first time we've used this word to refer to the normal "car" vehicle? I know SUMO uses "passenger". I want to make sure we are being consistent about our vehicle types.

Reply:
Is `car` from the [VEHICLE_CONFIGS](https://github.com/huawei-noah/SMARTS/blob/master/smarts/core/vehicle.py#L74)?  I think `car` is too general and should be replaced with `sedan` or something.